database/connection.go

```diff
@@ -17,3 +24,20 @@ func NewPool(url string) (*pgxpool.Pool, *sqlc.Queries) {
 
 	return pool, queries
 }
+
+func RunMigrations(databaseURL string) {
+	u, err := url.Parse(databaseURL)
+	if err != nil {
+		log.Fatal().Err(err).Msg("Invalid database URL")
+	}
+
+	db := dbmate.New(u)
+	db.FS = fs // Set the embedded filesystem for migrations
+	db.MigrationsDir = []string{"migrations"}
+	db.AutoDumpSchema = false
+
+	err = db.CreateAndMigrate()
+	if err != nil {
+		log.Fatal().Err(err).Msg("Error migrating")
```

Comment:
Same idea here, I don't know if `Err(err).Msg()` would actually work as intended, since zerolog is JSON logging. To elaborate, this may discard the error context and just send the message "Error migrating", which is unclear.

```suggestion
		log.Fatal().Err(err).Send()
```